### ISSUE TITLE
Separating /blog and marketing content from /docs

### ISSUE THREAD

**Issue by isaac-pulumi** [pulumi/docs#3194]
### Problem description
We missed (or nearly missed) two important marketing moments due to the docs build being either slow or broken:
1) We could not produce a viable preview of the launch day website until EoD the day before launch
2) It took >2 hours to update the Pulumi Up(date) page to include embedded video -- costing valuable impressions and leading to a sub-par user experience (a registration form for an event that had already finished)

### Affected product version(s)
All versions

### Suggestions for a fix
We need to partition /blog and other marketing pages (home, /about, /webinars, /superpowers, etc.) from /docs so that updating a marketing page does not get stuck waiting for all of /docs to rebuild. This is likely a first step in moving to a proper CMS.

**Comment by lukehoban** [comment]
@cnunciato @zchase Is this done in practice now?  At least to the level we have no intention to make further changes in the nearterm?

**Comment by cnunciato** [comment]
Yep, that's right -- we've made big improvements since this was submitted, and while we aren't planning any big changes as of now, the module-based approach we have not allows for continued incremental improvement. So I think we can call this done.

**Comment by cnunciato** [comment]
Or not. 😄
